src/coreclr/tools/superpmi/superpmi/commandline.cpp

```diff
@@ -110,6 +110,11 @@ void CommandLine::DumpHelp(const char* program)
     printf("     If 'workerCount' is not specified, the number of workers used is\n");
     printf("     the number of processors on the machine.\n");
     printf("\n");
+    printf(" -streaming filename\n");
+    printf("     Streaming mode. Read and execute work requests from indicated file (can be 'stdin').\n");
+    printf("     Each line is a method context number and additional force jit options for that method.\n");
+    printf("     Blank line or EOF terminates\n");
```

Comment:
Does streaming work with parallel? If not, can the parser below check that during validation?

The format of the streaming file needs to be better described (it doesn't mention `!` separation, for example).

It seems like the format maybe should be multi-line, with an empty line or something like `[streaming go]` kicking off the run. E.g.,
```
[streaming mc=12345] // start of streaming parameter block. Maybe mc could be a range? e.g., 1-2,4,6-70
JitRLCSE=
# try 0.02 (comment...)
JitRLCSEAlpha=0.02
JitRandomCSE=161
[streaming go]
```

Reply:
There could be lots more flexibility here, but the way I'm using it now I do not know very far in advance what the next run will look like, so there aren't opportunities for batching.

Reply:
Well, it's not about batching; it's about allowing multiple lines for configuration instead of one long single-line unreadable block of text.

Reply:
Currently all the streaming inputs are written by RLCSE so a compact format works well. I guess if streaming inputs are authored by hand then readability matters a bit more. How about something like:
```
REPLAY
METHODS
... (one or more lines of method numbers, ranges, etc)
OPTIONS
... (one option per line)
GO
(server carries out command and replies)
```
and later this can be expanded to handle more things, eg switching jits or collections or ...

Reply:
That format seems good to me. I wonder if the "commands" need to use magic characters to make sure the things between them can't be ambiguous, e.g. `:METHODS` or `[METHODS]`. That could be changed later if necessary.